### ISSUE TITLE
Sass compiling error

### ISSUE THREAD

**Issue by dmitrue** [tinymce/tinymce#9829]
Hi there,

Compiling tinymce with Sass giving the following error now:

<img width="642" alt="Screenshot 2024-08-17 at 12 28 24" src="https://github.com/user-attachments/assets/7ad63cf2-64c5-443d-86e0-7e6b358efd2c">

**Comment by lostkeys** [comment]
Hi @dmitrue. Here is the relevant source code: https://github.com/tinymce/tinymce/blob/24555233fc0531ee201fa438d327dd46822cfe3b/modules/oxide/src/less/theme/components/toolbar/toolbar.less#L20. 

TinyMCE doesnt use Sass, so I assume you meant LESS. What version of LESS are you using?

**Comment by dmitrue** [comment]
Hi @lostkeys! I'm using https://github.com/rails/cssbundling-rails with Bootstrap Sass

**Comment by lostkeys** [comment]
Alright, problem solved, you need to use a LESS compiler, not a Sass compiler. Here is the documentation for how to customize the skins: https://www.tiny.cloud/docs/tinymce/latest/creating-a-skin/

**Comment by dmitrue** [comment]
How is this solved? Not everyone using less. I don't need to customize skins, I'm only importing them because it doesn't work without it.

**Comment by lostkeys** [comment]
You are right sorry, I missunderstood the question. Can you help debug a bit and simply add `px` after the zero in the calc() so the line becomes `background-size: calc (100% - 11px * 2) calc (100% - 0px) ;` and recompile to see if that is what is tripping up the Sass compiler.

**Comment by dmitrue** [comment]
Thanks for reopening this issue. Yep, if I set `calc (100% - 0px);` everything compiles properly

**Comment by lostkeys** [comment]
Thanks for verifying! I'll arrange having that function always use a unit suffix in a future version of TinyMCE.

**Comment by petschki** [comment]
Same problem here with webpack and `style-loader`:

```
ERROR in ./node_modules/tinymce/skins/ui/oxide-dark/skin.css (./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./node_modules/tinymce/skins/ui/oxide-dark/skin.css)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
100% and 0 are incompatible.
     ╷
3864 │   background-size: calc(100% - 11px * 2) calc(100% - 0);
```

**Comment by ArvinJ-H** [comment]
Hi @dmitrue @petschki 

Thank you for reporting this issue. We're pleased to inform you that it has been addressed in TinyMCE version 7.4, which is now available.

To resolve the problem, please upgrade to the latest version of TinyMCE.
